cfme/utils/bz.py

```diff
@@ -78,16 +78,18 @@ def default_product(self):
 
     @classmethod
     def from_config(cls):
-        url = cfme_data.get("bugzilla", {}).get("url")
-        product = cfme_data.get("bugzilla", {}).get("product")
+        bz_conf = env.get('bugzilla', {})  # default empty so we can call .get() later
+        url = bz_conf.get('url')
         if url is None:
-            raise Exception("No Bugzilla URL specified!")
-        cr_root = cfme_data.get("bugzilla", {}).get("credentials")
-        username = credentials.get(cr_root, {}).get("username")
-        password = credentials.get(cr_root, {}).get("password")
-        return cls(
-            url=url, user=username, password=password, cookiefile=None,
-            tokenfile=None, product=product)
+            url = 'https://bugzilla.redhat.com/xmlrpc.cgi'
+            logger.warning("No Bugzilla URL specified in conf, using default: %s", url)
```

Comment:
This looks way cleaner just using an `or` or providing as default to the `.get()`, but this way we can have indication that we ignored or didn't find BZ URL.